Cubical/Foundations/Prelude.agda

```diff
@@ -169,6 +181,18 @@ compPathP-filler {A = A} {x = x} {B = B} p q j i =
                   (i = i1) → q j })
        (inS (p i)) j
 
+
+
+compPathP'-filler : ∀ {ℓ ℓ'} {A : Type ℓ} {B : A → Type ℓ'} {x y z : A} {x' : B x} {y' : B y} {z' : B z} {p : x ≡ y} {q : y ≡ z}
+             (P : PathP (λ i → B (p i)) x' y') (Q : PathP (λ i → B (q i)) y' z')
+          → PathP (λ j → PathP (λ i → B (compPath-filler p q j i)) x' (Q j)) P (compPathP' {x = x} {y = y} {x' = x'} {y' = y'} P Q)
+compPathP'-filler {B = B} {x' = x'} {p = p} {q = q} P Q j i =
+  fill (λ j → B (compPath-filler p q j i))
+        (λ j → λ { (i = i0) → x'  ;
+                    (i = i1) → Q j })
```

Comment:
Indentation. 

And why not define compPathP' in terms of this?

Reply:
I just wanted to be consistent with what is done with compPathP

Reply:
Ok, let's keep it as is and have a separate PR moving these and maybe redefining them

Reply:
compPathP and others were organized this way in https://github.com/agda/cubical/pull/258.